src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/Example1.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="InnerTypeLast"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.design.innertypelast;
+
+// xdoc section -- start
+class Test1 {
+  private String s;
+  class InnerTest1 {}
+  public void test() {} // violation
+}
+
```

Comment:
Why have you removed the explanation for the violation? 

```
// violation, method should be declared before inner types.
```

Reply:
Sorry about that....